lexicons/tools/ozone/history/getSubjectHistory.json

```diff
@@ -0,0 +1,40 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.history.getSubjectHistory",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Fetch the history of events for a specific subject.",
+      "parameters": {
+        "type": "params",
+        "required": ["subject"],
+        "properties": {
+          "subject": { "type": "string", "format": "uri" },
+          "limit": {
+            "type": "integer",
+            "minimum": 1,
+            "default": 50,
+            "maximum": 100
+          },
+          "cursor": { "type": "string" }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "events": {
+              "type": "array",
```

Comment:
Perhaps `events` will be required (and an empty array when there are no results).